### ISSUE TITLE
Establish actual passwords for users

### ISSUE THREAD

**Issue by jhannah** [jhannah/legislature_calendar#5]
Currently the password field is ignored. Actually implement users having passwords to control their accounts. Need password reset feature, which means we need to be able to send emails?

**Why?** Much of the value of this system (imo) derives from being logged into the system. Here's a view of "all users": http://72.14.177.244:8080/users Once you're logged into the system you can click the (W)atch / (Y)es / (N)o / (U)nwatch links. Doing so gives users a view of the bills they've clicked that they have interest in, so those bills go to the top of the list of ~1700 bills and they can, at a glance, see the updates / status of the bills they care about. Users can also see the list of bills other users care about. So if I, as a voter, care about the bills the League of Women Voters care about (for example), and their stance on those bills, I can see those bills at a glance. Above the list of all 1700 bills.

**Comment by barobba** [comment]
Yes, we will want to get user functionality working. Legiscan offers some, but we can augment with more. We might even want to consider "groups" and "group membership"... at some point.

I think for now, we should get the public information further along, then we can follow-up with logins.

**Comment by nitrocode** [comment]
@barobba suggested yesterday that we could use something like auth0

https://auth0.com/pricing

> Get Auth0 for free with up to 7,000 active users and unlimited logins. No credit card required.

https://auth0.com/docs/quickstart/webapp/golang/01-login
